internal/provider/testdata/DataSourceVMGroup_blank.cassette.yaml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Based on the `TestAccKatapultDataSourceVMGroup_blank` test, I don't think it would actually make any requests, so I think this might be a left-over cassette that can be removed :)